### ISSUE TITLE
Questions about where this is going

### ISSUE THREAD

**Issue by jpaas** [awslabs/aws-bootstrap-kit#78]
This is a really interesting tool. It literally encapsulates 2-3 years of learning and CDK development effort we have invested in at our company. As cool as it is, it's part of awslabs and appears not to  have any official kind of designation within AWS. Which is a little scary for anyone considering actually used it to bootstrap a project or a whole company. Is that wise/recommended? Where is AWS going with this? Will it keep up with the CDK and the rest of the ecosystem? Will it graduate to become official/supported? 

Worst case the plug gets pulled or the project gets abandoned, but we can always fork the open source I suppose.

I'm also wondering about the design intent. Is it meant to be loaded and fired once, or is it meant to be used to continually to evolve the organization, add new projects, etc?

**Comment by flochaz** [comment]
Hi @jpaas, Thanks for your interest and feedback.
At Amazon we have this principle of two ways doors decision and here we tried to make sure we (as maintainers) and you (as users) have this reversible choice:
It has been made open source to create a community help maintain it and make it evolve (and we have seen some success around this already) but as well to make sure, if we fail with the former, user could fork, migrate or other strategy to escape from it.

Anyway, for now there is a team behind it (not full time though as you may have noticed), and we are aggregating the numbers as we speak to evaluate where we want to go next.

Regarding your last question, it is the latter: git being your source of truth, any modification to your organization should be done through a modification of your SDLC organization project. At some point we would need to make sure we can manage more than accounts, and DNS, maybe automate SSO part, add [SCP management](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) ...

So the conclusion here is, we will spend the time to make our position and roadmap clearer in the following days.

Thanks again for your feedback !

**Comment by jpaas** [comment]
Thanks for the reply @flochaz, I look forward to seeing the roadmap.

One other question, perhaps the roadmap could explain. I didn't know about Control Tower until I got to the end of the tutorial. It also seems like a really useful and important service. The thing I've found myself wondering is ok, if I start with Control Tower, or I've migrated from aws-bootstrap-kit to Control Tower, now what? I'm just a bit unsure what the role of aws-bootstrap-kit is within a Control Tower environment. Do I still use it to deploy new accounts and then use Control Tower to manually enroll those accounts? Or should I be using the Control Tower account factory and there really isn't a role for aws-bootstrap-kit in that environment?

**Comment by flochaz** [comment]
Good point, we will make sure we mention Control Tower earlier in the doc.

AWS Bootstrap kit is meant to be a good starting point for a dev team with what we decided to be the minimal set of best practices for the cheapest price (by limiting the number services involved).

 But assuming you want to get to the next level, then two choices are open to you : 
* continue with a CDK based solution to manage your accounts, permissions etc.  by making evolve the bootstrap kit initial construct following your own requirements
* Move to a more complete tool such as Control Tower which as it's Pros and Cons (as described in the migration section) but then indeed, I would abandon BootstrapKit in favor of Control Tower since (as you may have noticed) the migration of accounts into this environment can be time consuming if done regularly (and can't be automated due to lack of APIs on Control Tower side ...)

**Comment by jpaas** [comment]
Thanks @flochaz that really helps
